sdk/python/feast/cli.py

```diff
@@ -773,6 +774,34 @@ def serve_registry_command(ctx: click.Context, port: int):
     store.serve_registry(port)
 
 
+@cli.command("serve_offline")
+@click.option(
```

Comment:
What is the purpose of the host parameter? 
what other values can we use apart from “0.0.0.0”? 
(there is no such option in the registry server, btw)

Reply:
I was thinking to connect to remote server by passing host

Reply:
Isn’t this the purpose of the client, instead? Otherwise I don’t understand who and  when that server starts

Reply:
I will remove that parameter if it's not required. When looking at  the feature `serve` command, it has a 'host' parameter, will remove that if it not required here https://github.com/RHEcosystemAppEng/feast/blob/master/sdk/python/feast/cli.py#L616